### PR TITLE
League stats

### PR DESCRIPTION
**Pull request by plapicola**
Adds count, offense, and defense methods for league statistics iteration.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -144,4 +144,56 @@ def group_games_by_season
     end
   end
 
+  def count_of_teams
```
Comment (TyMazey) [inline lib/stat_tracker.rb]: pshhh.

File: test/team_test.rb [hunk 2]
```diff
@@ -4,9 +4,9 @@ class TeamTest < Minitest::Test
 
   def setup
     attributes = {team_id: "1",
-                  franchiseId: "23",
```
Comment (TyMazey) [inline test/team_test.rb]: why u do this

File: test/team_test.rb [hunk 3]
```diff
@@ -4,9 +4,9 @@ class TeamTest < Minitest::Test
...
-                  shortName: "New Jersey",
-                  teamName: "Devils",
+                  franchiseid: "23",
```
Comment (TyMazey) [inline test/team_test.rb]: 👍

File: test/team_test.rb (hunk 2 above)
Comment (plapicola) [reply test/team_test.rb]: We needed to adjust the case of the symbols to handle the usage of the header_converter for reading in our CSV files.

**Pull request merged**